.github/workflows/nodejs.yml

```
@@ -1,5 +1,5 @@
 name: Build
-on: [push]
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
```

With this change we'll build on pushes to main and on PR's targeting main, which should reduce the amount of unhelpful CI runs we have.